File: front/src/components/company/dtos/update-auth_profile-dto.ts

import { CompanyAuth } from '../../company/dtos/company-auth';
import { CompanyProfile } from '../../company/dtos/company-profile';

export class Update_Auth_ProfileDto {
  companyAuthId!: number;
  companyProfileId!: number;
  companyAuth!: CompanyAuth;
  companyProfile!: CompanyProfile;
}

File: back/Application/CompaniesServices/Dtos/Update_Auth_ProfileDto.cs


using Application.CompaniesServices.Dtos.Auth;
using Application.CompaniesServices.Dtos.Profile;

namespace Application.CompaniesServices.Dtos;

public class Update_Auth_ProfileDto
{
    public required int CompanyAuthId { get; set; }
    public required int CompanyProfileId { get; set; }
    public required CompanyAuthDto CompanyAuth { get; set; }
    public required CompanyProfileDto CompanyProfile { get; set; }

}